template-helpers/dashboard.js

```diff
@@ -6,19 +6,27 @@ const { makeBreachCards } = require("./breaches");
 function getBreachesForEachEmail(args) {
   const verifiedEmails = args.data.root.verifiedEmails;
   const locales = args.data.root.req.supportedLocales;
+  let breachesFound = false;
 
   verifiedEmails.forEach(email => {
     const breachCards = makeBreachCards(email.breaches, locales);
     email.foundBreaches = {};
     email.foundBreaches.firstFourBreaches = breachCards.slice(0, 4);
     email.foundBreaches.remainingBreaches = breachCards.slice(5, breachCards.length);
```

Comment:
Unrelated, but I think this was causing a bug:
https://github.com/mozilla/blurts-server/issues/902#issuecomment-492004302

Not sure if this should be `.slice(4, breachCards.length);`